### PR TITLE
Update Yarn

### PR DESCRIPTION
**Pull request by Kipjr**

### CODE REVIEW

**Comment by SubSide** [comment]
@Kipjr Can't review the files (as the changes are too big or something?). Can you describe what exactly has changed?

**Comment by Kipjr** [comment]
> @Kipjr Can't review the files (as the changes are too big or something?). Can you describe what exactly has changed?


Everything:
- `https://registry.yarnpkg.com` --> `https://registry.npmjs.org`
- Quoting of each key & value

It seems the packages themselves have not yet been updated. Will look into this.

**Comment by Kipjr** [comment]
Was able to update it but unable to update these outdated packages
```
yarn outdated v1.22.19
Package            Current Wanted  Latest Package Type    URL

@types/js-cookie   2.2.7   2.2.7   3.0.2  dependencies    https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/js-cookie
axios              0.21.4  0.21.4  0.27.2 devDependencies https://axios-http.com

dotenv-webpack     7.1.1   7.1.1   8.0.1  devDependencies https://github.com/mrsteele/dotenv-webpack#readme

just-clone         5.1.0   5.1.0   6.1.1  dependencies    https://github.com/angus-c/just#readme

resolve-url-loader 3.1.4   3.1.4   5.0.0  devDependencies https://github.com/bholloway/resolve-url-loader/tree/v5/packages/resolve-url-loader
sass-loader        10.3.1  10.3.1  13.0.2 devDependencies https://github.com/webpack-contrib/sass-loader

ts-loader          8.4.0   8.4.0   9.3.1  devDependencies https://github.com/TypeStrong/ts-loader

uuid               3.4.0   3.4.0   9.0.0  dependencies    https://github.com/uuidjs/uuid#readme

vue                2.7.10  2.7.10  3.2.39 devDependencies https://github.com/vuejs/core/tree/main/packages/vue#readme

vue-loader         15.10.0 15.10.0 17.0.0 devDependencies https://github.com/vuejs/vue-loader

vue-router         3.6.5   3.6.5   4.1.5  dependencies    https://github.com/vuejs/router#readme

```

File: database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php [hunk 1]
```diff
@@ -14,12 +14,13 @@ class CreatePersonalAccessTokensTable extends Migration
     public function up()
     {
         Schema::create('personal_access_tokens', function (Blueprint $table) {
-            $table->bigIncrements('id');
-            $table->uuidMorphs('tokenable');
+            $table->id();
```
Comment (SubSide) [inline database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php:17]: Next time don't do this by changing a migration, but always do this by creating a new migration. This so forwards compatibility can be maintained. (For now it doesn't matter as we're still in the development state)

Comment (Kipjr) [reply database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php:17]: They did this xD

Comment (SubSide) [reply database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php:17]: It's our problem if it breaks on our end ;P
They probably changed it in their initializing project. So you only get this code on project creation.

**Pull request merged**